packages/block-editor/src/store/private-actions.js

```diff
@@ -39,21 +42,115 @@ export function __experimentalUpdateSettings(
 	settings,
 	{ stripExperimentalSettings = false, reset = false } = {}
 ) {
-	let cleanSettings = settings;
-	// There are no plugins in the mobile apps, so there is no
-	// need to strip the experimental settings:
-	if ( stripExperimentalSettings && Platform.OS === 'web' ) {
-		cleanSettings = {};
-		for ( const key in settings ) {
-			if ( ! privateSettings.includes( key ) ) {
-				cleanSettings[ key ] = settings[ key ];
+	return ( { dispatch, registry } ) => {
```

Comment:
I have to check this code more, but at first sight it doesn't seem to be in the right place..

Reply:
Yeah I know, I don't like it either. It would be great to find a better place for it :)